docs/framework/guides/deep-dive/localization.md

```diff
@@ -85,28 +85,34 @@ t( { string: '%1 %0 emoji', plural: '%1 %0 emojis', id: 'ACTION_N_EMOJIS' }, [ q
 This example shows how to create a localizable user interface of a plugin. Here is how you can create a button that will insert a smiling face emoji. The button will have a localizable tooltip.
 
 ```js
-// ...
-editor.ui.componentFactory.add( 'smilingFaceEmoji', locale => {
-	const buttonView = new ButtonView( locale );
-
-	// The translation function.
-	const { t } = editor.locale;
-
-	// The localized label.
-	const label = t( 'Insert smiling face emoji' );
-
-	buttonView.set( {
-		label,
-		icon: emojiIcon,
-		tooltip: true
-	} );
-
-	buttonView.on( 'execute', () => {
-		editor.execute( 'insertSmilingFaceEmoji' );
-		editor.editing.view.focus();
-	} );
-} );
-// ...
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
```

Comment:
I see the idea, which was to make this example full plugin without the need for ellipses, however, this requires more changes in the guide which are out of the scope of https://github.com/ckeditor/ckeditor5/issues/13201. 

I guess our safe option for now would be to add comments to those 2 ellipses without changing the whole example and, eventually, discuss later need for broader changes in this guide 🤔

Reply:
Sure, thanks for the explanation. I've added comments to the ellipses instead. :blush: